src/components/chatX/ChatXPage.js

```diff
@@ -70,12 +78,13 @@ export class ChatXPage extends React.Component {
     }
 
     render() {
+        const chatX = this.props.chatX;
         return (
             <div>
                 <div className="chat-x-header row">
                     <h1>Here you can chat with the other users</h1>
                 </div>
-                <ChatXBody messageList={this.state.messageList} messageCount={this.state.messageCount} />
+                <ChatXBody messageList={chatX.msgList} />
```

Comment:
Now I get messageList from redux state via the props

Reply:
👍